### ISSUE TITLE
[Accessibility] Keyboard shortcut key is unavailable to access the buttons(One Page, Two Page...)

### ISSUE THREAD

**Issue by Jessie-Zhang01** [dotnet/winforms#3682]
* .NET Core Version: 
.NET SDK 5.0.100-rc.1.20377.7

* Have you experienced this same bug with .NET Framework?: 
Yes

**More Info:**

1. This issue can also repro on Framework project.
2. There is a similar bugs: [1156785 ](https://dev.azure.com/devdiv/DevDiv/_workitems/edit/1156785?src=WorkItemMention&src-action=artifact_link)in VSO.
3. We can just use ‘Enter’ key enter to buttons(One Page, Two Page...).

**Minimal repro:**
1. Extract “[WindowsFormsApp3.zip](https://github.com/dotnet/winforms/files/5005021/WindowsFormsApp3.zip)” and open it in VS.
2. Build and run the project.
3. Using Tab key to navigate to Preview button, then press Enter key to show Print preview dialog.
4. Check whether any shortcut key is provided to buttons(One Page, Two Page...) to access.

**Expected behavior:**
Keyboard shortcut key should be available to access the buttons(One Page, Two Page...).

**Actual behavior:**
Keyboard shortcut key is unavailable to access the buttons(One Page, Two Page...).

**Comment by Tanya-Solyanik** [comment]
Perhaps we can use (ShortcutKeys)[https://docs.microsoft.com/en-us/dotnet/api/system.windows.forms.toolstripmenuitem.shortcutkeys?view=netcore-3.1] property with these buttons.

**Comment by SergeySmirnov-Akvelon** [comment]
Hi @Jessie-Zhang01. I have investigated this issue and i see two cases:
- First case - user opens PrintPreviewDialog. In this case user can get access to buttons using "Tab" key.
- Second case - user clicks "Enter" button and focus disappears. This behavior already is fixed in #3598.
![3682](https://user-images.githubusercontent.com/23376742/92609459-43e55600-f2bf-11ea-82e0-c785d01c6c1c.gif)
@Jessie-Zhang01, probably we should close it as duplicate #3598. Could you please check that it is not reproduced for last 5.0/master?

**Comment by Jessie-Zhang01** [comment]
In fact, this issue wants to have some separate shortcut keys to open One page, Two pages, etc. For example, press Ctrl + o to open ‘One page’ that I am thinking of. Here is the comment from @merriemcgaw "I think this is its own issue where we need to add Shortcut keys to the dialog."
In addition, [3598 ](https://github.com/dotnet/winforms/issues/3598)is fixed on the latest 5.0 RC2.

**Comment by RussKie** [comment]
I'd probably go with <kbd>CTRL</kbd>+<kbd>1</kbd> for "one page", <kbd>CTRL</kbd>+<kbd>2</kbd> for "two page", etc. through to <kbd>CTRL</kbd>+<kbd>5</kbd>, if we'd go down this route.

**Comment by merriemcgaw** [comment]
I like @russkie's idea. Will that work @Jessie-Zhang01 ?

**Comment by Jessie-Zhang01** [comment]
@merriemcgaw there is no any shortcut key that we can used to access the buttons(One Page, Two Page...). So it would be a good idea to use RussKia's method.

**Comment by merriemcgaw** [comment]
@SergeySmirnov-Akvelon, looks like we've got consensus on how to do this one. Please add the new shortcuts and make sure they show up in the ToolTip, and Narrator announces.

**Comment by Jessie-Zhang01** [comment]
Verified the issue with latest NET SDK 6.0.100-alpha.1.20557.2, this issue is fixed. The user will be able to switch between viewing modes using the shortcut keys (Ctrl+1, Ctrl+2, etc) on PrintPreviewDialog.
![step1](https://user-images.githubusercontent.com/45864649/98518307-2d04b500-22aa-11eb-9e79-a130c66eff3a.gif)
